### DISCUSSION TITLE
Install termux repo playstore version to f-droid/github version

### DISCUSSION THREAD

**Discussion by JeelsBoobz** [termux/termux-packages#20960] [Q&A] [answered]
Any solution to install termux repo playstore version into f-droid or github version?

**Comment by sylirre** [comment, marked as answer]
You can't install Play Store Termux version over F-Droid or GitHub. They are signed with different keys and Android OS won't let you to install over.

Installation: https://github.com/termux/termux-app?tab=readme-ov-file#installation

And regarding to Google Play Store, remember that it is not supported by current maintainers team. Report all issues of Google Play app to https://github.com/termux-play-store/termux-apps.

Termux on Google Play isn't newer or has more features than officially released https://github.com/termux/termux-app/releases/tag/v0.118.1 (stable) or https://github.com/termux/termux-app/releases/tag/v0.119.0-beta.1 (beta).
